### ISSUE TITLE
CUDA invalidates Revise

### ISSUE THREAD

**Issue by timholy** [timholy/Revise.jl#746]
Pointed out by @KristofferC

**Comment by benz0li** [comment]
@timholy *Invalidating* as described in issue https://github.com/timholy/Revise.jl/issues/701?

**Comment by KristofferC** [comment]
No, invalidating as in having to recompile some Revise functions after CUDA is loaded.
